<launch>
    <include file="$(find gazebo_ros)/launch/empty_world.launch">
        <arg name="world_name" value="$(find turtlebot3_gazebo)/worlds/empty.world"/>
        <arg name="paused" value="false"/>
        <arg name="use_sim_time" value="true"/>
        <arg name="gui" value="true"/>
        <arg name="headless" value="false"/>
        <arg name="debug" value="false"/>
    </include>

    <include file="$(find rlss_ros)/launch/turtlebot_sim.launch" ns="turtlebot1">
        <arg name="robot_idx" value="1"/>
        <arg name="robot_description_path" value="/home/baskin/rlss_ws/src/rlss-ros/third_party/rlss/examples/inputs/demo_descriptions/3d/hetero/turtlebot_1.json"/>
        <arg name="x_pos" value="-3.51892"/>
        <arg name="y_pos" value="1.88924"/>
        <arg name="z_pos" value="0.25"/>
        <arg name="robot_frame" value="turtlebot1/base_footprint"/>
        <arg name="parent_frame" value="turtlebot1/odom"/>
        <arg name="commander_dt" value="0.05"/>
    </include>


    <include file="$(find rlss_ros)/launch/turtlebot_sim.launch" ns="turtlebot2">
        <arg name="robot_idx" value="2"/>
        <arg name="robot_description_path" value="/home/baskin/rlss_ws/src/rlss-ros/third_party/rlss/examples/inputs/demo_descriptions/3d/hetero/turtlebot_2.json"/>
        <arg name="x_pos" value="2.29582118084"/>
        <arg name="y_pos" value="-0.685492687181"/>
        <arg name="z_pos" value="0.25"/>
        <arg name="robot_frame" value="turtlebot2/base_footprint"/>
        <arg name="parent_frame" default="turtlebot2/odom"/>
        <arg name="commander_dt" value="0.05"/>
    </include>
</launch>